comment/utils.py

```diff
@@ -206,3 +207,24 @@ def get_username_for_comment(comment):
             return comment.email.split('@')[0]
         return settings.COMMENT_ANONYMOUS_USERNAME
     return comment.user.username
+
+
+def validate_order():
+
+    @lru_cache()
+    def get_allowed_orders():
```

Comment:
Can you please move this method outside. And please avoid using nested functions except for docerators.

Reply:
the method `get_allowed_orders` isn't required anywhere else in the codebase. To avoid exposing it anywhere else, it is used here. Such types of closures are even used in the official python codebase.

Using classes here seems an overkill here to me.